### ISSUE TITLE
A11y: Label for the select all checkbox

### ISSUE THREAD

**Issue by martinfrances107** [jsdrupal/drupal-admin-ui#458]
For the MultipleFields widgets the number of INPUT elements is dynamic.

In terms of HTML DOM structure there is a standard pattern for this case

```
<fieldset>
  <legend> 
    <input/> 
    <input/> 
     ...
   </legend>
</fieldset>
```
 
Where are almost there.... the PR will be only a few lines of adjustment.

**Comment by dawehner** [comment]
Nice point!

**Comment by martinfrances107** [comment]
Coming back to looking at this after a little break..

My original drive was to clear a critical accessibility bug. [ "critical" as seen by running "REACT_APP_AXE=true yarn start" and looking at the console logs.] 

A) The structure as I see it today does not have the accessibility concerns that warrant adjusting the HTML structure. So I am just going to leave well alone

B) The select all checkbox is missing the requisite aria-label or equivalent. 

So I am going to redirect this issue and provide a one line fix.

**Comment by martinfrances107** [comment]
Thanks for the merge.
